docs/resources/dcs_instance.md

```diff
@@ -81,125 +83,202 @@ resource "huaweicloud_dcs_instance" "instance_1" {
 
-* `begin_at` - (Required, String) Time at which backup starts. "00:00-01:00" indicates that backup starts at 00:00:00.
+* `period_type` - (Optional, String) Interval at which backup is performed. Default value is `weekly`.
+  Currently, only weekly backup is supported.
 
-* `period_type` - (Required, String) Interval at which backup is performed. Currently, only weekly backup is supported.
+* `backup_at` - (Required, List) Day in a week on which backup starts, the value ranges from 1 to 7.
+  Where: 1 indicates Monday; 7 indicates Sunday.
 
-* `backup_at` - (Required, List) Day in a week on which backup starts. Range: 1–7. Where: 1 indicates Monday; 7
-  indicates Sunday.
+* `begin_at` - (Required, String) Time at which backup starts.
+  Format: `hh24:00-hh24:00`, "00:00-01:00" indicates that backup starts at 00:00:00.
 
 ## Attributes Reference
 
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - A resource ID in UUID format.
-* `vpc_name` - Indicates the name of a vpc.
-* `subnet_name` - Indicates the name of a subnet.
-* `security_group_name` - Indicates the name of a security group.
-* `resource_spec_code` - Resource specification code.
+
+* `status` - Cache instance status. The valid values are as follows:
+  + `RUNNING`: The instance is running properly.
+    Only instances in the Running state can provide in-memory cache service.
+  + `ERROR`: The instance is not running properly.
+  + `RESTARTING`: The instance is being restarted.
+  + `FROZEN`: The instance has been frozen due to low balance.
+    You can unfreeze the instance by recharging your account in My Order.
+  + `EXTENDING`: The instance is being scaled up.
+  + `RESTORING`: The instance data is being restored.
+  + `FLUSHING`: The DCS instance is being cleared.
+
+* `domain_name` - Domain name of the instance. Usually, we use domain name and port to connect to the DCS instances.
+
+* `max_memory` - Total memory size. Unit: MB.
+
 * `used_memory` - Size of the used memory. Unit: MB.
-* `internal_version` - Internal DCS version.
-* `max_memory` - Overall memory size. Unit: MB.
-* `user_id` - Indicates a user ID.
-* `user_name` - Username.
-* `ip` - Cache node's IP address in tenant's VPC.
-* `port` - Port of the cache node.
+
+* `vpc_name` - The name of VPC which the instance belongs to.
+
+* `subnet_name` - The name of subnet which the instance belongs to.
+
+* `security_group_name` - The name of security group which the instance belongs to.
+
+* `order_id` - The ID of the order that created the instance.
+
+## Timeouts
+
+This resource provides the following timeouts configuration options:
+
+* `create` - Default is 20 minute.
+* `update` - Default is 20 minute.
+* `delete` - Default is 15 minute.
+
+## Import
+
+DCS instance can be imported using the `id`, e.g.
+
+```sh
+terraform import huaweicloud_dcs_instance.instance_1 80e373f9-872e-4046-aae9-ccd9ddc55511
+```
```

Comment:
please refer to the [import](https://registry.terraform.io/providers/huaweicloud/huaweicloud/latest/docs/resources/compute_instance#import) session in compute_instance.

Reply:
done